src/rules/latin-var-name-only-rule/latin-var-name-only.js

```diff
@@ -0,0 +1,39 @@
+const notLatinSymbolRegexp = /[^\w$]/;
+
+const CYRILLIC_TO_LATINA_CHAR = require('../latin-var-name-only-rule/cyrillic-to-latina-char');
+
+
+module.exports = {
+	meta: {
+		type:'problem',
+		fixable:'code'
+	},
+	create (context) {
+		return {
+			Identifier (node) {
+				let hasVarNameNonLatinSymbol = notLatinSymbolRegexp.test(node.name);
+				if (hasVarNameNonLatinSymbol) {
+					context.report({
+						node, 
+						message: 'Variable name consist non-latin char',
+						fix: (fixer) => {
+							let member = node.name;
+
+							let memberCharArray = member.split('');
+
+							let fixedObjectMemberCharArray = memberCharArray.map((char) => {
```

Comment:
I recommend you to define a separate `cyrillicToLatinChar` function and import it instead of object